### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by dieum512**
In this branch we:

1. Updated the HTML file
2. updated the CSS file
3. Created a js file
4. added some js functionality to the menu hamburger with js
5. updated the readme file and 
6. corrected some linters errors

### CODE REVIEW

**Review by Owusu-Desmond — APPROVED** [review]
Hi @dieum512  and @fauziyahzeeyah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅
📌 I love your design ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -272,7 +277,7 @@ <h3 class="connect-tittle">LET’S CONNECT</h3>
           <h2>Languages</h2>
           <i class="fas fa-chevron-down"></i>
         </div>
-        <ul class="languages-list"  aria-label=" Programming languages" >
+        <ul class="languages-list" aria-label=" Programming languages">
           <li>
             <img src="images/Ellipse-1.png" alt="javaScript logo" />
             <h4>JavaSript</h4>
```
Comment (cindykandie) [inline index.html:283]: Kindly avoid using headings in the middle of the elements such as ul/li to allow accessibility for users using screen readers, instead try using p or span

**Comment by batoorsohail** [comment]
# Highlights:
✔️ I love your code it is very clean 

# Optional suggestions:
- I think it would be better if you use button tag instead of a tag 😄

**Comment by UsamaMasood12** [comment]
You have done a good job. One suggestion is to try to make a space between your sections, it will look easier for the reader.

**Pull request merged**